templates/search.html

```diff
@@ -52,7 +52,10 @@ <h1 id="logo"><a href="/">feverbase</a></h1>
         <span id="filters-status">{{ 'Hide' if adv_filters_in_use else '' }}</span>
         Advanced Filters
       </a>
-      <br>
+      <br />
+      <a onclick="resetAdvancedFilters()">
+        <span id="filters-reset">{{ 'Reset Advanced Filters<br />' if adv_filters_in_use else '' }}</span>
```

Comment:
don't mix HTML with the text and just make #filters-reset a div if you're looking for `block` rendering